docs/04-operation-guides/operations/10-backup-kyma.md

```diff
@@ -25,10 +25,9 @@ Taking volume snapshots is possible thanks to [Container Storage Interface (CSI)
 
 You can create on-demand volume snapshots manually, or set up a periodic job that takes automatic snapshots periodically.
 
-## Back Up Resources Using Velero
+## Back Up Resources Using 3rd Parties Tools
 
-You can back up and restore individual resources manually or automatically with Velero. For more information, read the [Velero documentation](https://velero.io/docs/).
-Be aware that a full backup of a Kyma cluster isn't supported. Start with the existing Kyma installation and restore specific resources individually.
+3rd part tols like velero backup is not supported at this time.
```

Comment:
```suggestion
You can back up and restore individual resources manually or automatically using third-party tools, such as [Velero](https://velero.io/docs/).

> [!WARNING] 
> These tools may have limitations and not support fully automated cluster backups.
> Additionally, they often require specific access rights to your infrastructure provider, which may not be available in Kyma's managed offerings where you lack sufficient access rights to the infrastructure account.
```

Reply:
I suggest using the content provided by Marco Porru.

Reply:
I have updated the content as below.
```
Third-party tools like Velero Backup are not currently supported. These tools may have limitations and might not fully support automated cluster backups. They often require specific access rights to cluster infrastructure, which may not be available in Kyma's managed offerings, where access rights to the infrastructure account are restricted
```

Reply:
@IwonaLanger  Please check the contents now.

Reply:
@AribaOpensource In the context of the Open Source project (and Kyma Project refers to the Open Source project) Velero is supported. It's not applicable for SAP BTP, Kyma runtime, because the hyperscaler account is managed by SAP/Kyma team.

For this reason, I would prefer to stay on the version provided in my comment.